src/watcher/watcher.ts

```diff
@@ -43,9 +43,14 @@ export function createWatcher(ctx: Context) {
   let includePaths: Array<RegExp> = [];
   let ignorePaths: Array<RegExp> = [];
 
+  // ensure root is string[]
+  const root = typeof props.root === "string" ? [props.root] : props.root
+  // ensure roots are absolute paths
+  const absRoots = root && root.map(r => path.resolve(`${env.SCRIPT_PATH}`, r));
```

Comment:
let's use `ensureScriptRoot` 
this will allow users to use abs paths

Reply:
Ah, cool, didn't know that was there, I'll change that.

By the way, a cool feature of `path.resolve()` is that it already handles abs paths... 

![image](https://user-images.githubusercontent.com/158330/77642945-e047b600-6f34-11ea-94fa-318d36d721bd.png)